style.css

```diff
@@ -303,3 +327,159 @@ input {
   color: #42526e;
   margin-left: 25px;
 }
+
+/* ---------------Mobile Vesrion Ends Here--------------- */
+
+/* -----------------------Desktop Vesrion Started----------------------- */
+
+@media screen and (min-width: 768px) {
+  #headline-img-desktop {
+    display: block;
+    width: 100%;
+  }
+
+  .navbar {
+    display: block;
+    margin-left: 41%;
+  }
+
+  #navbar-list {
+    list-style-type: none;
+  }
+
+  #navbar-list li {
+    display: inline;
+    padding: 12px;
+  }
+
+  a:link {
+    text-decoration: none;
+  }
+
+  #logo-image,
+  .menu,
+  #headline-img-mobile {
+    display: none;
+  }
+
+  .text {
+    padding-bottom: 10%;
+  }
+
+  #links-list {
+    margin-right: 8%;
+  }
+
+  #action {
+    margin-left: 29%;
+  }
+
+  .second-text {
+    width: 91%;
+    height: auto;
+  }
+
+  .cards-group {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    row-gap: 30px;
+    justify-content: center;
+  }
+
+  #Recent-Works-Text {
+    width: 100%;
+    height: auto;
+    padding-left: 2%;
+  }
+
+  #about-text {
+    padding-right: 345px;
+  }
+
+  .about-me {
+    width: 100%;
+    display: grid;
+    row-gap: 30px;
+    justify-content: center;
+  }
+
+  #get-my-resume-btn {
+    margin-left: 27%;
+  }
+
+  .about-detail {
+    width: 900px;
+    height: 96px;
+  }
+
+  .about1 {
+    display: none;
+  }
+
+  .skills {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    justify-content: center;
+  }
+
+  .form-section {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  #form-detail {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    margin-top: 5%;
+    padding-left: 7%;
+  }
+
+  .form-description {
+    width: 100%;
+  }
+
+  #form-desc {
+    width: 75%;
+    margin-left: 13%;
+    text-align: justify;
+  }
+
+  .form {
+    width: 50%;
+  }
+
+  #fname,
+  #email {
+    width: 400px;
+    height: 48px;
+  }
+
+  #textarea {
+    width: 400px;
+    height: 114px;
+    border-radius: 4px;
+    border-color: #60c095;
+  }
+
+  #get-in-btn {
```

Comment:
- [ ] Like the previous reviewer mentioned kindly give all your `buttons` interactions. 🙏 . Apologies for the wrong screenshot given by the previous reviewer

![image](https://user-images.githubusercontent.com/74814780/193282674-9601a3d1-414a-4625-b378-74c0e2114ace.png)

Reply:
Hover, and Pressed interactions are added for buttons.